certora/helpers/MockWadRayMath.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "../../src/math/WadRayMath.sol";
+
+contract MockWadRayMath {
```

Comment:
Those contracts already exist in the test suite I guess you can reuse them no?

Reply:
Yes I even think that we should remove them from the tests (they are only used for the gas report, which is not very useful)

Reply:
You mean replace them for gas snapshots?

Reply:
yes ! It works much better

Reply:
I wouldn't say better because it's not testing the same thing at all. One allows you to check that your change is improving gas efficiency or not while the other is to compare gas from the reference which is a different goal IMO

Reply:
We can agree on getting rid of them, but we should be sure to understand what we're removing first

Reply:
We could compare to reference with snapshot also